packages/plugin-reconstruction/src/index.spec.ts

```diff
@@ -7,7 +7,7 @@ jest.useFakeTimers();
 
 describe("reconstruction", () => {
   test("default starting value renders correctly", async () => {
-    let timeline = [
+    const timeline = [
       {
         type: reconstruction,
         stim_function: function (val) {
```

Comment:
Arrow function with implicit return? :)

Reply:
Sorry, I'm dense 😁 Is there something I should do differently with this one?

Reply:
My fault, I was reviewing only your latest commit and couldn't comment on lines unrelated to your changes...
I was just after
```ts
        stim_function: (val) => `<p>${Math.round(val * 10)}</p>`,
```